saleor/graphql/views.py

```diff
@@ -181,6 +186,11 @@ def handle_query(self, request: HttpRequest) -> JsonResponse:
             span.set_tag("http.useragent", request.headers.get("user-agent", ""))
             span.set_tag("span.type", "web")
 
+            source_service_name = get_source_service_name_value(
+                request.headers.get("source-service-name")
```

Comment:
This header will be blocked by CORS because it's not part of `access-control-allow-headers`.

Reply:
fix: https://github.com/saleor/saleor/pull/17285